### ISSUE TITLE
moment.locale() :: Uncaught ReferenceError :: When browser's languages array is passed as argument.

### ISSUE THREAD

**Issue by lots0logs** [moment/moment#2856]
The documentation says that `moment.local()` accepts an array of locale strings from which it will use the first match it finds. However, when I try to use `window.navigator.languages` (which is an array of locale strings) as the argument I get an Uncaught ReferenceError. 
```javascript
moment().locale(window.navigator.languages);
```
```javascript
Uncaught ReferenceError: module is not defined(…)
loadLocale @ moment-with-locales.min.js:267
chooseLocale @ moment-with-locales.min.js:249
locale_locales__getLocale @ moment-with-locales.min.js:340
locale_locales__getSetGlobalLocale @ moment-with-locales.min.js:287
(anonymous function) @ VM417:2
```

 This occurs with the latest version of `moment.js` downloaded from `momentjs.com` (http://momentjs.com/downloads/moment-with-locales.js)

**Comment by Till86** [comment]
I got a simular issue, for me it crashes saying, that module (line 6 in the min file) can not be resolved if i parse a locale. So it thinks I'm Italien.

**Comment by fexdev** [comment]
I got the same after an update today. in Line 267 in moment-with-locales.js it changed from `typeof module !== 'undefined'`to `!isUndefined(module)`. But in my Browser-Context, `module`doesn´t exist and I think that´s why it´s not working. After changing the line back it´s working again.

**Comment by lynchem** [comment]
+1 - I set the locale in startup 

moment.locale('en-gb');

and it starting throwing that same error when I updated.

**Comment by lots0logs** [comment]
Fixed in #2865

**Comment by mj1856** [comment]
@lots0logs - Please verify fixed with 2.11.1 (which contains #2865)
